demos/Dawn.mscx

```diff
@@ -10,25 +10,154 @@
+      <fingeringFontFace>Edwin</fingeringFontFace>
+      <lhGuitarFingeringFontFace>Edwin</lhGuitarFingeringFontFace>
+      <rhGuitarFingeringFontFace>Edwin</rhGuitarFingeringFontFace>
+      <stringNumberFontFace>Edwin</stringNumberFontFace>
+      <longInstrumentFontFace>Edwin</longInstrumentFontFace>
+      <longInstrumentFontSize>10</longInstrumentFontSize>
+      <shortInstrumentFontFace>Edwin</shortInstrumentFontFace>
+      <shortInstrumentFontSize>10</shortInstrumentFontSize>
+      <partInstrumentFontFace>Edwin</partInstrumentFontFace>
+      <partInstrumentFontSize>14</partInstrumentFontSize>
+      <dynamicsFontFace>Edwin</dynamicsFontFace>
       <dynamicsFontSize>11</dynamicsFontSize>
+      <expressionFontFace>Edwin</expressionFontFace>
+      <expressionFontSize>10</expressionFontSize>
+      <expressionOffset x="0" y="2.5"/>
+      <tempoFontFace>Edwin</tempoFontFace>
+      <metronomeFontFace>Edwin</metronomeFontFace>
       <metronomeFontSize>10</metronomeFontSize>
+      <measureNumberFontFace>Edwin</measureNumberFontFace>
       <measureNumberFontSize>9</measureNumberFontSize>
+      <measureNumberFontStyle>2</measureNumberFontStyle>
+      <translatorFontFace>Edwin</translatorFontFace>
       <translatorFontSize>10</translatorFontSize>
+      <systemFontFace>Edwin</systemFontFace>
+      <staffFontFace>Edwin</staffFontFace>
+      <staffPosAbove x="0" y="-1"/>
+      <staffPosBelow x="0" y="2.5"/>
+      <rehearsalMarkFontFace>Edwin</rehearsalMarkFontFace>
+      <repeatLeftFontFace>Edwin</repeatLeftFontFace>
+      <repeatRightFontFace>Edwin</repeatRightFontFace>
+      <frameFontFace>Edwin</frameFontFace>
+      <textLineFontFace>Edwin</textLineFontFace>
+      <textLineFontSize>10</textLineFontSize>
+      <glissandoFontFace>Edwin</glissandoFontFace>
+      <bendFontFace>Edwin</bendFontFace>
+      <headerFontFace>Edwin</headerFontFace>
       <headerFontSize>9</headerFontSize>
+      <headerFontStyle>1</headerFontStyle>
+      <footerFontFace>Edwin</footerFontFace>
+      <footerFontSize>9</footerFontSize>
+      <instrumentChangeFontFace>Edwin</instrumentChangeFontFace>
+      <stickingFontFace>Edwin</stickingFontFace>
       <stickingFontSize>10</stickingFontSize>
+      <user1FontFace>Edwin</user1FontFace>
+      <user2FontFace>Edwin</user2FontFace>
+      <user3FontFace>Edwin</user3FontFace>
+      <user4FontFace>Edwin</user4FontFace>
+      <user5FontFace>Edwin</user5FontFace>
+      <user6FontFace>Edwin</user6FontFace>
+      <user7FontFace>Edwin</user7FontFace>
+      <user8FontFace>Edwin</user8FontFace>
+      <user9FontFace>Edwin</user9FontFace>
+      <user10FontFace>Edwin</user10FontFace>
+      <user11FontFace>Edwin</user11FontFace>
+      <user12FontFace>Edwin</user12FontFace>
+      <letRingFontFace>Edwin</letRingFontFace>
+      <palmMuteFontFace>Edwin</palmMuteFontFace>
+      <fermataPosAbove x="0" y="-0.5"/>
+      <fermataPosBelow x="0" y="0.5"/>
+      <trillMinDistance>0.5</trillMinDistance>
```

Comment:
As for these, I don't know why 3.6 Beta added them... It happens for other scores opened in 3.6 Beta too, sometimes they are added, sometimes they are deleted...